README.md

```diff
@@ -124,6 +124,20 @@ This section is a place for additional best practices that may be useful but are
 }
 ```
 
+## How do I disable auto fix
+
+You may wish to disable autofix so when you run `eslint` in ci it will error
```

Comment:
consistency nit, choose one of the following

```suggestion
## How do I disable autofix

You may wish to disable autofix so when you run `eslint` in ci it will error
```

```suggestion
## How do I disable auto fix

You may wish to disable auto fix so when you run `eslint` in ci it will error
```